File: java/spring-hibernate/src/main/java/com/sprint/sms/api/bean/dummy/Dummy3.java
package com.sprint.sms.api.bean.dummy;

import com.ideahut.common2.annotation.IdhFormatter;

@IdhFormatter
public class Dummy3
{
	private String id;
	
	private String name;

	@IdhFormatter
	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	@IdhFormatter
	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
	
}

File: php/slim-doctrine/app/src/com/sprint/sms/api/bean/dummy/Dummy3.php
<?php
namespace App\com\sprint\sms\api\bean\dummy;

use \App\com\sprint\sms\api\support\FormatSupport;

class Dummy3 implements FormatSupport
{
	private $id;
	private $name;
	
	public function setId($id) {
		$this->id = $id;
	}
	
	public function getId() {
		return $this->id;
	}
	
	public function setName($name) {
		$this->name = $name;
	}
	
	public function getName() {
		return $this->name;
	}
	
	public function toFormatObject() {
		$o = array();
		$o['id'] = $this->id;
		$o['name'] = $this->name;		
		return $o;
	}
	
}